### PR TITLE
Feature/dx 1068 sidebar exclude

### PR DESCRIPTION
**Pull request by NullableInt**
Uses the pre-exisiting `sidebar_hidden` to hide pages from the sidebar

### CODE REVIEW

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- _plugins/sidebar.rb  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5771817&bid=18891646)

File: _plugins/sidebar.rb [hunk 1]
```diff
@@ -16,12 +16,14 @@ def initialize
...
       menu_order = page['menu-order'].nil? ? 0 : page['menu-order']
+      sidebar_hidden = page['hide_from_sidebar'].nil? ? false : page['hide_from_sidebar']
       url = page['url'].gsub('index.html', '').gsub('.html', '')
       @hash_pre_render[url] = {
         title: page['title'],
         url: page['url'].gsub('.html', ''),
         name: page['name'],
-        menu_order: menu_order
+        menu_order: menu_order,
+        sidebar_hidden: sidebar_hidden
```
Comment (asbjornu) [inline _plugins/sidebar.rb:26]:
```suggestion
        sidebar_hidden: hide_from_sidebar
```

File: _plugins/sidebar.rb [hunk 2]
```diff
@@ -106,6 +108,7 @@ def render(filename)
       merged = merge(@hash_pre_render, @filename_with_headers)
       merged.select { |key, _value| key.split('/').length <= 2 }.sort_by { |_key, value| value[:menu_order] }.each do |key, value|
         next if value[:title].nil?
+        next if value[:sidebar_hidden]
```
Comment (asbjornu) [inline _plugins/sidebar.rb:111]:
```suggestion
        next if value[:hide_from_sidebar]
```

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- _plugins/sidebar.rb  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5771817&bid=18891646)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- _plugins/sidebar.rb  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5771817&bid=18891646)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- _plugins/sidebar.rb  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5771817&bid=18891646)

**Comment by NullableInt** [comment]
@asbjornu I can't merge because there are 1681 errors reported in the verify step.

**Pull request merged**